cmake/BTApplication.cmake

```diff
@@ -28,6 +28,11 @@ ENDIF()
 
 FIND_PACKAGE(Sword 1.8.1 REQUIRED)
 
+# Translation related
+FIND_PACKAGE(PkgConfig REQUIRED)
+FIND_PACKAGE(Gettext REQUIRED)
+FIND_PACKAGE(IsoCodes REQUIRED)
```

Comment:
We have to make use of `IsoCodes_PREFIX`, and might also have to check whether `IsoCodes_DOMAINS` contains the necessary domains.